rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -349,7 +349,7 @@
               "$ref": {
                 "title": "MEF person reference",
                 "type": "string",
-                "pattern": "^https://mef.rero.ch/api/mef/.*?$",
+                "pattern": "^https://mef.rero.ch/api/.*/.*?$",
```

Comment:
Is it possible to have a more specific rule here, something like `gnd or idref or rero or bnd`?

Reply:
Actualy you can have mef, viaf, bnf, gnd, idref, rero

Reply:
Schould I replace `.*` with `bnd\|gnd\|idref\|rero`?